src/lib/seo/ai.robots.txt

```diff
@@ -0,0 +1,42 @@
+User-agent: AI2Bot
```

Comment:
Not available as npm package, so added a download script instead. Could run the script as a `prep:ai-robots-txt` and gitignore this file. But unlike the other `prep:` scripts the `download-ai-robots-txt.ts` script fetches a resource from a source outside our control. So opted to check this file into git. If a `prep:` script is preferred I can live with that :)